app/lib/bk/compat/parsers/circleci/orbs/docker/build.rb

```diff
@@ -0,0 +1,58 @@
+# frozen_string_literal: true
+
+module BK
+  module Compat
+    module CircleCISteps
+      # Handles the trasnlation of docker/build command
+      class DockerOrb
+        def translate_docker_build(config)
+          [
+            ('export DOCKER_BUILDKIT=1' if config['use-buildkit']),
+            ("# Using BuildKit context at #{config['attach-at']}" if config['attach-at']),
+            (if config['lint-dockerfile']
+               '# Dockerlint is not supported at this time and should be translated by hand'
+             end),
+            *pull_cache_images(config),
+            generate_build_command(config)
+          ].compact
+        end
+
+        private
+
+        def pull_cache_images(config)
+          config.fetch('cache_from', '').split(',').map do |image|
+            "docker pull #{image.strip} || true"
+          end
+        end
+
+        def generate_build_command(config)
+          [
+            'docker',
+            ('buildx' if config['cache_to']),
+            'build',
+            '-f', File.join(config.fetch('path', '.'), config.fetch('dockerfile', 'Dockerfile')),
+            '-t', generate_tag(config),
+            ("--build-context source=#{config['attach-at']}" if config['attach-at']),
```

Comment:
taking into account the `attach-at` option appears to be specific to CircleCI (documentation states that it will mount a workspace previously saved with `persist_to_workspace`, which is not supported) I'm not sure that it is a good idea to translate it this way, but I'm not 100% against it either